scripts/adopt-a-dog.js

```diff
@@ -0,0 +1,3 @@
+function clickHandler(str) {
+    alert(str);
```

Comment:
You are on the right track. For one, you'll want two functions. One for clicking the adopt button, the other for clicking on the image of the dog. Also, you'll want some sort of variable outside of the functions to keep track of the cost. Maybe: `let totalCost = 0` For the adopt button, you'll want something like `func adoptDog(cost, breed, name)` Then in your function, you'd alert the user of the dog, and add the cost to the totalCost and display that. Reach out during class or come by office hours if you'd like some more help